src/data/reducers/portalConfiguration.js

```diff
@@ -1,49 +1,32 @@
 import {
-  FETCH_PORTAL_CONFIGURATION_REQUEST,
-  FETCH_PORTAL_CONFIGURATION_SUCCESS,
-  FETCH_PORTAL_CONFIGURATION_FAILURE,
+  SET_PORTAL_CONFIGURATION,
   CLEAR_PORTAL_CONFIGURATION,
 } from '../constants/portalConfiguration';
 
 const initialState = {
-  loading: false,
-  error: null,
   enterpriseId: null,
+  enterpriseName: null,
   enterpriseSlug: null,
   enterpriseLogo: null,
 };
 
 const portalConfiguration = (state = initialState, action) => {
   switch (action.type) {
-    case FETCH_PORTAL_CONFIGURATION_REQUEST:
+    case SET_PORTAL_CONFIGURATION:
       return {
         ...state,
-        loading: true,
-        error: null,
-      };
-    case FETCH_PORTAL_CONFIGURATION_SUCCESS:
-      return {
-        ...state,
-        loading: false,
-        enterpriseId: action.payload.data.enterprise_customer,
-        enterpriseSlug: action.payload.data.enterprise_slug,
-        enterpriseLogo: action.payload.data.logo,
-      };
-    case FETCH_PORTAL_CONFIGURATION_FAILURE:
-      return {
-        ...state,
-        loading: false,
-        error: action.payload.error,
-        enterpriseId: null,
-        enterpriseSlug: null,
-        enterpriseLogo: null,
+        enterpriseId: action.payload.data.uuid,
+        enterpriseName: action.payload.data.name,
+        enterpriseSlug: action.payload.data.slug,
+        enterpriseLogo: action.payload.data.branding_configuration
```

Comment:
This is split onto multiple lines for linting purposes (line exceeds max length otherwise).